DSharpPlus/Enums/Interaction/InteractionResponseState.cs

```diff
@@ -0,0 +1,22 @@
+namespace DSharpPlus;
+
+/// <summary>
+/// Represents the state of an interaction regarding responding.
+/// </summary>
+public enum InteractionResponseState
```

Comment:
should be in DSharpPlus/Entities (with that as its namespace) and the name should be prefixed with Discord-

Reply:
The entire folder is scuffed, then

Reply:
correct, i will be migrating it once we have less open PRs that would be scuffed by that diff

Reply:
Also should it really be prefixed with Discord, since this is, in fact, an ad-hoc enum and does not reflect Discord's entity?

Reply:
> The entire folder is scuffed, then

(Every enum in the folder is under DSharpPlus, unless I'm somehow out of date)
![image](https://github.com/DSharpPlus/DSharpPlus/assets/42438262/ee5946f2-da39-4b5d-9ef6-f9e042ecff18)

Reply:
not all enums are in the folder :v

Reply:
Disregard, communicating on GH is very ![](https://cdn.discordapp.com/emojis/1211822332688080956.gif?size=24&quality=lossless)

Reply:
> Also should it really be prefixed with Discord, since this is, in fact, an ad-hoc enum and does not reflect Discord's entity?
impl detail of the type - for all the user knows it may not even be related to discord's entities, so, yes